sources/packages/web/src/views/institution/locations/offerings/LocationOfferingView.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Study period offerings"
+        :routeLocation="routeLocation"
+        subTitle="View Request"
+      >
+      </header-navigator>
```

Comment:
There is a banner to say, if the offering has existing applications.

![image](https://github.com/bcgov/SIMS/assets/54600590/0e178dc2-ed15-4b29-bed6-bf5177796e72)